oak_ml_transparency/mnist/README.md

```diff
@@ -0,0 +1,109 @@
+# The MNIST example
+
+This folder contains scripts for building an MNIST model and evaluating it
+against adversarial examples. The build script is only included here to make the
+example self-contained. The eval script should work with any MNIST model in
+[Tensorflow SavedModel format](https://www.tensorflow.org/guide/saved_model).
+
+## Building the Docker image
+
+We use a Docker image, with all the required dependencies installed to build the
+model, and run the evaluation. Having such a Docker image is necessary for
+achieving transparency. The Dockerfile should include the script for running the
+evaluation. More specifically, with the following lines in the Dockerfile, we
+embed the evaluation script in the Docker image:
+
+```bash
```

Comment:
This is Docker, not bash (not sure if there is syntax for it though)

Reply:
Changed it to `dockerfile`, as it is a [Language supported by Github](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml).